### DISCUSSION TITLE
Strings werden in verkehrter Reihenfolge angezeigt

### DISCUSSION THREAD

**Discussion by musbach** [tbnobody/OpenDTU#2074] [General]
Hallo,
ich habe an den HMT-2250 fünf Module an die Anschlüsse 1 bis 5 (von oben nach unten) angeschlossen, der 6. Anschluss ist frei (siehe Screenshot aus der Bedienungsanleitung). In den Live-Daten werden mir aber die Strings 2 bis 6 angezeigt. Der 1. String ist leer. Ist die Reihenfolge einfach nur vertauscht?

![Strings (2)](https://github.com/tbnobody/OpenDTU/assets/20500637/f42b996f-adf5-483d-9a74-972cec07123c)

![Strings (1)](https://github.com/tbnobody/OpenDTU/assets/20500637/2a0d8a6a-06dc-489a-adb5-7efbd0be7648)

**Comment by tbnobody** [comment]
Bitte lese #1025

**Comment by musbach** [comment]
Schönes Durcheinander ;)
